docs/src/examples/components/Form/Usage/FormExampleCheckbox.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react'
+import { Form, Button, Checkbox, Provider } from '@stardust-ui/react'
+
+const fields = [
+  {
+    label: 'Email',
+    name: 'email',
+    id: 'email-inline-shorthand',
+    key: 'email',
+    required: true,
+    inline: true,
+  },
+  {
+    control: {
+      as: Checkbox,
+      labelPosition: 'start',
```

Comment:
```suggestion
```

It's not required, so let's omit it

Reply:
I think it looks better, when the form's structure is unified, i.e. description on the left, user's input on the right.

Reply:
We can go whatever you will decide, but should mention:
- that default position is on left across Web:
  - https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_input_type_checkbox
  - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox
- we should keep examples minimal and avoid questions like "`labelPosition` is required to use this component"?